gateway/uplink.go

```diff
@@ -31,6 +31,27 @@ func (g *gateway) parseDataUplink(ctx context.Context, phyPayload []byte) (strin
 		return "", map[string]interface{}{}, errNoDevice
 	}
 
+	g.logger.Debugf("received uplink from %s", device.NodeName)
+
+	// we will send one device downlink from the do command per uplink.
+	if len(device.Downlinks) > 0 {
+		g.logger.Debugf("sending downlink message to %s", device.NodeName)
+		payload, err := g.createDownlink(device, device.Downlinks[0])
+		if err != nil {
+			return "", map[string]interface{}{}, fmt.Errorf("failed to create downlink: %w", err)
+		}
+
+		err = g.sendDownlink(ctx, payload, false, packetTime)
+		if err != nil {
+			return "", map[string]interface{}{}, fmt.Errorf("failed to send downlink: %w", err)
+		}
+
+		g.logger.Infof("sent the downlink packet to %s", device.NodeName)
```

Comment:
Is this noisy?

Reply:
this could be noisy once we support confirmed data, but we can maybe wait until that pr to evaluate